VocabularyTrainer/AddWord/AddNewWordViewController.swift

```diff
@@ -210,12 +220,24 @@ final class AddNewWordViewController: UIViewController {
         UserDefaults.standard.set(vocabularies, forKey: language)
         UserDefaults.standard.set(vocabulariesSuccessRates, forKey: languageVocabProgressKey)
         UserDefaults.standard.set(vocabulariesAddDates, forKey: languageVocabDateAddedKey)
-        
         resetConfigs()
         self.delegate?.wordAdded()
         showToast(message: NSLocalizedString("New word added", comment: "New word added"), yCoord: 340.0)
     }
     
+    @objc private func addNewWordTapped() {
+        hasDuplicates ? showAlert() : addWordToDictionary()
+    }
+    
+    func showAlert() {
+        let duplicateWordAlert = UIAlertController(title: "Duplicate Word Detected", message: "This word already exists in the word list", preferredStyle: .alert)
```

Comment:
Please use `NSLocalizedString` for all the strings.

I think it makes sense to include the word itself in the alert's message as well. Also we should mention that an existing word will be overwritten when using tap anyway.

Can you change it to something like:
"The word XYZ already exists. Adding the new pair will overwrite the old pair of words."